server/command.go

```diff
@@ -323,3 +324,65 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 	p.postCommandResponse(args, fmt.Sprintf("Unknown action %v", action))
 	return &model.CommandResponse{}, nil
 }
+
+func getAutocompleteData() *model.AutocompleteData {
+	github := model.NewAutocompleteData("github", "[command]", "Available commands: connect, disconnect, todo, subscribe, unsubscribe, me, settings")
+
+	connect := model.NewAutocompleteData("connect", "", "Connect your Mattermost account to your GitHub account")
+	private := model.NewAutocompleteData("private", "", "If used, the github bot will ask for read access to your private repositories")
+	connect.AddCommand(private)
+	github.AddCommand(connect)
+
+	disconnect := model.NewAutocompleteData("disconnect", "", "Disconnect your Mattermost account from your GitHub account")
+	github.AddCommand(disconnect)
+
+	todo := model.NewAutocompleteData("todo", "", "Get a list of unread messages and pull requests awaiting your review")
+	github.AddCommand(todo)
+
+	subscribe := model.NewAutocompleteData("subscribe", "[owner/repo] [features] [flags]", "Subscribe the current channel to receive notifications about opened pull requests and issues for an organization or repository. [features] and [flags] are optional arguments")
+	subscribe.AddTextArgument("Owner/repo to subscribe to", "[owner/repo]", "")
+	subscribe.AddTextArgument("(Optional) Comma-delimited list of one or more of: issues, pulls, pushes, creates, deletes, issue_comments, pull_reviews, label:\"<labelname>\". Detaults to pulls,issues,creates,deletes", "[features]", `/[^,-\s]+(,[^,-\s]+)*/`)
+	flags := []model.AutocompleteListItem{{
+		HelpText: "events triggered by organization members will not be delivered (the organization config should be set, otherwise this flag has not effect)",
+		Hint:     "(optional)",
+		Item:     "--exclude-org-member",
+	}}
+	subscribe.AddStaticListArgument("Currently supports --exclude-org-member", false, flags)
```

Comment:
Does it make sense to show this option, if no organisation is configured?

Reply:
Good question.. I'm not aware of any way in the Autocomplete API to show a suggestion conditional on the current state of the plugin. Would you suggest I turn this field into a text field so users must manually type the flag they would like to use, and therefore this option will not be automatically filled?